PianoFromAbove/GameState.cpp

```diff
@@ -988,18 +945,18 @@ GameState::GameError MainScreen::MsgProc( HWND hWnd, UINT msg, WPARAM wParam, LP
     return Success;
 }
 
-GameState::GameError MainScreen::Logic( void )
+GameState::GameError MainScreen::Logic(void)
 {
     // Start new ImGui frame
     m_pRenderer->ImguiStartFrame();
 
-    static Config &config = Config::GetConfig();
-    static PlaybackSettings &cPlayback = config.GetPlaybackSettings();
-    static const ViewSettings &cView = config.GetViewSettings();
-    static const VisualSettings &cVisual = config.GetVisualSettings();
-    static const VideoSettings &cVideo = config.GetVideoSettings();
-    static const VizSettings &cViz = config.GetVizSettings();
-    const MIDI::MIDIInfo &mInfo = m_MIDI.GetInfo();
+    static Config& config = Config::GetConfig();
+    static PlaybackSettings& cPlayback = config.GetPlaybackSettings();
+    static const ViewSettings& cView = config.GetViewSettings();
+    static const VisualSettings& cVisual = config.GetVisualSettings();
+    static const VideoSettings& cVideo = config.GetVideoSettings();
+    static const VizSettings& cViz = config.GetVizSettings();
+    const MIDI::MIDIInfo& mInfo = m_MIDI.GetInfo();
```

Comment:
I know you were working without git for a while, but if you want to PR something to someone's project, please try to keep formatting changes out of it (or at least separate it into a separate commit). It makes it a lot harder to pick out actual code changes and leads to unnecessary merge conflicts down the line.